lib/batchVersions.js

```diff
@@ -139,7 +139,7 @@ export async function ensureBatchVersion({tokenizerId}) {
  *
  * @returns {object} A tokenizerVersion object.
```

Comment:
This returns something different when `explain` is passed -- so that should be documented.

Reply:
I went ahead an updated this. Not sure if this is exactly what you were looking for but let me know.